src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -266,12 +266,25 @@ def post(self, request: Request, group, integration_id) -> Response:
             )
 
         installation = integration.get_installation(organization_id=organization_id)
-        try:
-            data = installation.create_issue(request.data)
-        except IntegrationFormError as exc:
-            return Response(exc.field_errors, status=400)
-        except IntegrationError as e:
-            return Response({"non_field_errors": [str(e)]}, status=400)
+
+        with ProjectManagementEvent(
+            action_type=ProjectManagementActionType.CREATE_EXTERNAL_ISSUE, integration=integration
+        ).capture() as lifecycle:
+            lifecycle.add_extras(
+                {
+                    "provider": integration.provider,
+                    "integration_id": integration.id,
+                }
+            )
+
+            try:
+                data = installation.create_issue(request.data)
```

Comment:
might be worth tracking with a different slug b/c i have a feeling we have different levels of success based on where the ticket is created from.

also wondering if we should have a SLO around the endpoint that allows the user to search for a ticket in the 3p in the issue details page.

Reply:
Yeah I think you're right 🤔 I'll add that now.